bio_hansel/kmer_count/__init__.py

```diff
@@ -7,6 +7,7 @@
 import logging
 import pandas as pd
 
+from bio_hansel.quality_check import perform_quality_check
```

Comment:
`from ..quality_check import perform_quality_check` to be consistent with other import statements and to allow base package name to change (not that it will)